### DISCUSSION TITLE
Is there a way to know if a form's changes were saved

### DISCUSSION THREAD

**Discussion by simplecommerce** [vazco/uniforms#1105] [Q&A] [answered]
Hi,

I was wondering if there is a way to know when a form's changes were saved.

For example: 

I have a form with autosave on, every time I make a change, the form is submitted.
Is there a state that will indicate to me that the current changes were saved?

**Comment by wadamek65** [comment, marked as answer]
Hi @simplecommerce , apologies for the late response.

Unfortunately, nothing comes to my mind that you could use out of the box. The `submitted` variable will only tell you whenever something was submitted regardless of whether it succeeded or not. 

A possible solution to your problem could be extending the Form that you are using with a custom one and overriding the `onSubmit` method to set an additional flag within state that you could then pass onto your context.

Would such a solution help solve your problem?

**Reply by simplecommerce** [reply]
No problem, thanks for the response, that is what I also though, I just wanted to see if you guys had any other suggestions, that is exactly what I did on my end at the moment, thanks again!!
